infrastructure/ansible/roles/dataset_loader/tasks/dataset_loader.yml

```diff
@@ -102,7 +102,7 @@
       name: "{{ item.name }}"
       description: "{{ item.description }}"
       useInTable: "{{ item.useInTable }}"
-  with_items: "{{ dl_ds_merged_types }}"
+  with_items: "{{ dl_ds_private_types }}"
```

Comment:
This should still use `dl_ds_merged_types`.

Reply:
updated